cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -173,6 +180,16 @@ function Download($download_url, $platforms) {
     Write-Verbose "  Unpacked $bin_name"
     $bin_paths += "$tmp\$bin_name"
   }
+
+  if ($null -ne $info["updater"]) {
+    $updater_id = $info["updater"]["artifact_name"]
+    $updater_url = "$download_url/$updater_id"
+    $out_name = "$tmp\{{ app_name}}-update.exe"
+
+    $wc.downloadFile($updater_url, $out_name)
+    $bin_paths += $out_name
+  }
```

Comment:
do we want to add printouts for "unpacked updater"?

Reply:
The list of binaries will already include a thing that ends in `-update`, but I could add an extra message on top of that if you think it'd help legibility.